website/src/scripts/header.js

```
@@ -0,0 +1,125 @@
+      d.querySelector('.nav-dropdown-trigger')?.setAttribute('aria-expanded', 'false');
+    });
+  });
+
+  /* ── Burger + Drawer ────────────────────────── */
+  const burger = document.getElementById('burger-btn');
+  const drawer = document.getElementById('nav-drawer');
+  const drawerClose = document.getElementById('drawer-close');
+  const drawerScrim = document.getElementById('drawer-scrim');
+
+  function openDrawer() {
+    document.body.classList.add('drawer-open');
```

ESLint

[error] 'document' is not defined.

(no-undef)